vendors/circom/circomlib/json/groth16_proof.h

```diff
@@ -3,6 +3,7 @@
 
 #include "circomlib/json/json_converter_forward.h"
 #include "circomlib/json/points.h"
+#include "tachyon/base/logging.h"
```

Comment:
how does this relate to uniting circle point to affine points?

Reply:
This changes comes from a different change, I'll move this change into different commit

Reply:
@chokobole @ashjeong I think it is still unreflected.